locales/en-US/webextension.properties

```diff
@@ -16,6 +16,8 @@ notificationLinkCopiedTitle = Link Copied
 # modifier key used in the shortcut (do not translate it): for example, Ctrl-V
 # on Windows systems.
 notificationLinkCopiedDetails = The link to your shot has been copied to the clipboard. Press {meta_key}-V to paste.
+# This is displayed as a warning when a Full Page save will be cropped
+imageCroppedWarning = This image has been cropped to {pixels} pixels.
```

Comment:
Should the string be, "This image _will be_ cropped to {pixels} pixels" ?

Reply:
If you're using "pixel", you should use a proper plural form. The easy way out is to use px.
http://projectfluent.io/fluent/guide/selectors.html

Also CC @stasm to make sure we don't break anything if you go for proper plural.

Reply:
@flodolo In this case the singular will never be needed, so I'm not sure I see the need for the extra selector syntax

Reply:
@johngruen said "px", but I thought that encouraged a term that a normal human wouldn't know so I switched to pixels.

Reply:
Once again, stop using your English brain to think ;-)

one vs many is not a plural form, it's something that works in a bunch of languages
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Use_proper_plural_forms